lib/uri-beacon-scanner.js

```diff
@@ -63,29 +63,25 @@ UriBeaconScanner.prototype.parseUriBeacon = function(peripheral) {
 
   var flags = data.readUInt8(0); // flag is the 1st byte
   var txPower = data.readInt8(1); // TX Power is 2nd byte
-  var uriData = data.slice(2); // remainder in the URI
-
+  var firstByte = data[2];
+  var uriData = data.slice(3); // remainder in the URI
   var uriPrefix = '';
```

Comment:
My understanding of the [spec](https://github.com/google/uribeacon/blob/master/specification/AdvertisingMode.md#uribeacon-uri-scheme-prefix) was that you had to use Uri Scheme Prefix for it to be valid.
Would it be clearer if I changed the 'firstByte' variable to 'uriPrefixID'?